src/search.js

```diff
@@ -1,9 +1,20 @@
 const map = require("./map");
 
-module.exports = (code, comp) => {
-  const list = require(`../assets/${code}`);
-  return list.reduce((a, v) => {
-    v = map(v);
+module.exports = (data, comp) => {
+  let isCompresed = false;
```

Comment:
Typo: `isCompressed`

Reply:
I think, that the better way is to remove `isCompressed` at all. And create `.compress` function for  third party datasets.